data/json/npcs/missiondef.json

```diff
@@ -711,5 +711,26 @@
       "success_lie": "OK, then hand it over.",
       "failure": "Shit happens."
     }
+  },
+  {
+    "id": "MISSION_GET_CITY_COP_MOMENTO",
+    "type": "mission_definition",
+    "name": "Break into armory to retrieve family photo",
+    "goal": "MGOAL_FIND_ITEM",
+    "difficulty": 2,
+    "value": 800,
+    "item": "family_photo",
```

Comment:
you could a start function here and a bit of mapgen_update to guarantee that the family photo only spawns after the player accepts the quest.  It's probably not necessary given that there is a 100% chance the item will spawn, though.